server.js

```diff
@@ -20,7 +37,42 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('This is a Happy thoughts API.')
+})
+
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec()
+  res.json(thoughts)
+})
+
+app.post('/thoughts', async (req, res) => {
+  //Retrieve the information sent by the client to our API endpoint
+  const {message} = req.body
+
+  //Use our mpngoose model to create the database entry
+  const thoughts = new Thought({message})
+
+  try {
+    //Success - the thought is saved to the databse
+    const savedThought = await thoughts.save()
```

Comment:
Be mindful of the naming of your constant. You are creating 1 thought, `const thought` would have been more pertinent